docs/cookbook/customization/controller.rst

```diff
@@ -0,0 +1,35 @@
+Customizing Controllers
+=======================
+
+All **Sylius** resources are using the `` Sylius\Bundle\ResourceBundle\Controller\ResourceController`` as default. but some of them have been already extended in Bundles.
+If you want to override some controller action check which controller you should be extending.
+
+Why would you customize a Controller?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+How to customize a Controller?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create a new Controller class under the ``AppBundle/Repository`` namespace.
+Remember that it has to extend a proper base class. How can you check that?
+
+For the ``ProductController`` run:
+
+.. code-block:: bash
+
+    $ php app/console container:debug | grep sylius.controller.product
```

Comment:
You can and should simply run ``php app/console debug:container sylius.controller.product.